roles/kubernetes/node/templates/kubelet.j2

```diff
@@ -13,7 +13,7 @@ KUBELET_HOSTNAME="--hostname-override={{ ansible_hostname }}"
 --pod-infra-container-image={{ pod_infra_image_repo }}:{{ pod_infra_image_tag }} \
 --kube-reserved cpu={{ kubelet_cpu_limit }},memory={{ kubelet_memory_limit|regex_replace('Mi', 'M') }} \
 --node-status-update-frequency={{ kubelet_status_update_frequency }} \
-{% if kube_version | version_compare('v1.6', '>=') %}
+{% if kube_version | version_compare('v1.7', '>=') %}
 --enable-cri={{ kubelet_enable_cri }} --cgroups-per-qos={{ kubelet_cgroups_per_qos }} \
```

Comment:
sure. but i'm not a maintainer, so it has to go through review by one of them anyhow.